smdebug/tensorflow/keras.py

```diff
@@ -64,7 +64,15 @@ def _is_not_supported(self):
             if tf.executing_eagerly() or (
                 hasattr(self.model, "run_eagerly") and self.model.run_eagerly
             ):
-                pass
+                if is_tf_version_2x():
+                    self.logger.info(
+                        "Executing in TF2.x eager mode."
+                        "TF 2.x eager doesn't provide gradient and optimizer variable values."
+                        "SageMaker Debugger will not be saving gradients and optimizer variables in this case"
+                    )
+                else:
+                    self.logger.info("Disabling SMDebug as it does not support eager mode")
```

Comment:
for version < 2.0 , is there debug mode. this mesg is misleading.

Reply:
there is eager mode for < 2.0. it has to be explicitly enabled using enable_eager_execution() API. this log can be changed to mention does not support eager mode for 1.x (until it is confirmed)

Reply:
updated in the latest commit